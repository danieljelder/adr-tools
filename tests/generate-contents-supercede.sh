adr new First Decision
adr new -s 1 Second Decision
adr generate toc -p foo/doc/adr/
