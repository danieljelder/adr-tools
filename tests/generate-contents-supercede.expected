adr new First Decision
doc/adr/0001-first-decision.md
adr new -s 1 Second Decision
doc/adr/0002-second-decision.md
adr generate toc -p foo/doc/adr/
# Architecture Decision Records

* [~~1. First Decision~~](foo/doc/adr/0001-first-decision.md) _(Superceded by [2. Second Decision](foo/doc/adr/0002-second-decision.md))_
* [2. Second Decision](foo/doc/adr/0002-second-decision.md)
